### PR TITLE
Added a licence and some info to the README. 

### PR DESCRIPTION
**Pull request by Shy**

### CODE REVIEW

File: document_scanner_webinar/README.MD [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Getting started
+=====
+
+### Requirements
+
+To deploy this project you'll need accounts for the following services:
+
+- [Superblocks](https://superblocks.com/)
+
+### Setup
```
Comment (jshatsky22) [inline document_scanner_webinar/README.MD]: I'd rather this be on the main README or we just point to the docs. Feels weird to have it inside this specific example

Comment (Shy) [reply document_scanner_webinar/README.MD]: It should be in all of them. You never know where SEO is gonna send you.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,21 @@
 # custom-components-examples
 
+![Video of Custom Components](screenshots/customComponents.gif)
```
Comment (jshatsky22) [inline README.md]: The GIF is pretty low quality. I think let's replace with the main hero image from the website. It's here: 

https://assets-global.website-files.com/627d359d0e0aa25158817bf0/64ef667a238938b55c05e99e_CustomComponents_HeroImage_FINAL.png

**Review by jshatsky22 — COMMENTED** [review]
Left some minor feedback and looks like there are merge conflicts to handle

**Review by jshatsky22 — APPROVED** [review]

**Pull request merged**